react-native-appboy-sdk.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = package['name']
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.source         = { :git => 'git+https://github.com/Appboy/appboy-react-sdk.git', :tag => s.version }
+
+  s.requires_arc   = true
+  s.platform       = :ios, '8.0'
+
+  s.preserve_paths = 'LICENSE.md', 'README.md', 'package.json', 'index.js'
+  s.source_files   = 'iOS/**/*.{h,m}'
+
+  s.dependency 'React'
```

Comment:
Per https://github.com/adjust/react_native_sdk/pull/14, could we make `Appboy-iOS-SDK` a dependency here instead of `React`, and update the comment in the Podfile accordingly (e.g., either use `pod Appboy-iOS-SDK` or `pod react-native-appboy-sdk`)?

Reply:
Both suggestions sound good. Will do.